benefits/templates/google_sso/login_sso.html

```diff
@@ -0,0 +1,37 @@
+{% comment %}
+    This file overrides the django-google-sso pacakge's login_sso.html template
```

Comment:
😇 
```suggestion
    This file overrides the django-google-sso package's login_sso.html template
```

Reply:
once you land something like https://github.com/cal-itp/calitp.org/issues/513 here i'll be out of a job. 🙃

Reply:
Fixed in https://github.com/cal-itp/benefits/pull/3270/commits/ec112a172a405f99cd0710e31a0f1e81c1beacb3